.gitignore

```diff
@@ -1 +1,2 @@
-node_modules
\ No newline at end of file
+node_modules
+coverage
```

Comment:
The coverage folder isnt included in the project,  so you can remove it from the gitignore